introduction/CAPMC_reinit_and_config.md

```diff
@@ -0,0 +1,78 @@
+# CAPMC Reinit and Configuration Notice
+
+CAPMC is now capable of doing a reinit of hardware that does not support
+GracefulRestart or ForceRestart. This is done by powering off the target
+component and then powering it back on. This behavior is enabled by default and
+controlled by a setting in the CAPMC configmap. Along with reinit is the
+ability to control other aspects of CAPMCs behavior. To do this, the CAPMC
+configmap must be modified to include a CapmcConfiguration section.
+
+## Instructions
+
+1. Edit the configmap
+
+```
+ncn-m001# kubectl edit configmaps -n services cray-capmc-configuration
+```
+At the end of the config.toml section, after the lines
```

Comment:
New substep.

2. Modify the CAPMC ConfigMap to include a CapmcConfiguration section.

    Navigate to the end of the config.toml section and look for the following lines:
    
    ```
        # Administratively defined maximum allowable system power consumption,
        # specified in watts
        PowerBandMax = 0
    ```
    
    After these lines, add the following section:
    
    ```
        [CapmcConfiguration]
        # Number of workers that are available to execute in parallel for Redfish calls
        # ActionMaxWorkers = 1000
        # CAPMC behavior for a power action that target hardware does not support
        # Valid options: simulate, ignore, error
        #   simulate - For components that do not support GracefulRestart or
        #              ForceRestart, simulate will turn the node Off then On again
        #   ignore - Skip the component but notify the user it was ignored
        #   error - Halt the power operation and notify the user
        # OnUnsupportedAction = "simulate"
        # CAPMC will check power state of components when an Off request has been
        # issued. CAPMC will return from the Off request when it has verified that the
        # target components are off or if the number of retries have been exceeded.
        # WaitForOffRetries = 4
        # Amount of time to sleep between checks of component power state for Off.
        # WaitForOffSleep = 15
    ```
    
    To change a setting, uncomment the line and modify the setting to a valid value.